src/modules/crud.js

```diff
@@ -0,0 +1,177 @@
+
+// Edit Tasks
+const editItems = (oldItem) => {
+  const newInput = document.createElement('input');
+  newInput.type = 'text';
+  newInput.classList.add('new-input');
+  newInput.value = oldItem.textContent;
+  oldItem.replaceWith(newInput);
+  const local = localStorage.getItem('items');
+  const data = JSON.parse(local);
+  newInput.addEventListener('keypress', (e) => {
+    const newId = newInput.parentElement.id;
+    if (e.key === 'Enter') {
+      const oldItem = document.createElement('div');
+      oldItem.textContent = newInput.value;
+      newInput.replaceWith(oldItem);
+      const editedContainers = document.querySelectorAll('.new-task');
+      for (let i = 0; i < editedContainers.length; i += 1) {
+        if (editedContainers[i].id === newId) {
+          data[i].description = newInput.value;
+          localStorage.setItem('items', JSON.stringify(data));
+        }
+      }
+    }
+  });
+};
+
+// Remove Tasks
+const removeItems = (li) => {
+  listOfTasks.removeChild(li);
+  let count = 1;
+  const parsedItems = localStorage.getItem('items');
+  let localData = JSON.parse(parsedItems);
+  // Filter tasks that are true
+  taskArray = JSON.parse(localStorage.getItem('items'));
+  taskArray.splice(li.id - 1, 1);
+  localData = taskArray;
+  // Update index of tasks
+  localData.map((item) => {
+    item.index = count;
+    count += 1;
+    return null;
+  });
+  localStorage.setItem('items', JSON.stringify(localData));
+};
+
+// Add new task
+const addNewTask = (newDescription) => {
+  const newTask = document.createElement('div');
+  newTask.classList.add('list');
+  newTask.innerHTML += `
+<input type="checkbox" class="checkbox">
+<span>${newDescription}</span>
+<span class="ellipsis">
+<i class="fa-solid fa-ellipsis-vertical"></i>
+</span>
+<span class="trashcan">
+<i class="fa-solid fa-trash-can"></i>
+</span>
+`;
+  listOfTasks.appendChild(newTask);
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((checkboxInput) => {
+    checkboxInput.addEventListener('click', () => {
+      checkboxInput.nextElementSibling.classList.toggle('strike');
+      checkboxInput.parentNode.classList.toggle('clicked-on');
+      const ellipsisIcon = checkboxInput.parentNode.childNodes[5];
+      ellipsisIcon.classList.toggle('inactive-ellipsis');
+      const trashIcon = checkboxInput.parentNode.childNodes[7].childNodes[1];
+      trashIcon.classList.toggle('active-trash');
+      completeItem();
+    });
+  });
+  // Create and send new task to local storage
+  const newListItem = new Task(newDescription, false, checkbox.length);
+  taskArray.push(newListItem);
+  const stringedItems = JSON.stringify(taskArray);
+  localStorage.setItem('items', stringedItems);
+  // Edit task
+  const editItem = document.querySelectorAll('.ellipsis');
+  editItem.forEach((item) => {
+    item.addEventListener('click', () => {
+      item.parentNode.classList.add('clicked-on');
+      editItems(item.previousElementSibling);
```

Comment:
You are violating the **DRY** principle which is not to repeat your code. Kindly check you are repeating the same code two times. If you have any doubt you may check out this article [DRY, KISS and YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md)